test/blackbox-tests/test-cases/pkg/opam-var/dune

```diff
@@ -0,0 +1,25 @@
+(cram
+ (deps opam-var-helpers.sh)
+ (applies_to :whole_subtree))
+
+(cram
+ (deps opam-vars)
+ (applies_to opam-var-os))
+
+(data_only_dirs fake_opam_root)
+
+; We trick opam into thinking it has a config file with fake_opam_root so that we can
+; access the os variables without having to do opam init
+
+(rule
+ (target opam-vars)
+ (deps fake_opam_root/config)
+ (action
+  (with-stdout-to
+   %{target}
+   (progn
```

Comment:
I experimented with making this `concurrent` and then sorting however `opam var` locks the directory so concurrent queries error. We would also had to bump our dune project version.